modules/tensor_mechanics/src/materials/FiniteStrainCrystalPlasticity.C

```diff
@@ -738,8 +751,9 @@ FiniteStrainCrystalPlasticity::solveStress()
   {
     dpk2 = - jac.invSymm() * resid; // Calculate stress increment
     _pk2_tmp = _pk2_tmp + dpk2; // Update stress
+    calcResidual(resid);
+    internal_variable_update_NRiteration(); //update _fe_prev_inv
```

Comment:
This function does not follow our naming conventions (member function should be camel-cased)